doc/classes/Transform3D.xml

```diff
@@ -5,7 +5,9 @@
 	</brief_description>
 	<description>
 		A 3×4 matrix (3 rows, 4 columns) used for 3D linear transformations. It can represent transformations such as translation, rotation, and scaling. It consists of a [member basis] (first 3 columns) and a [Vector3] for the [member origin] (last column).
+		?? The [Transform3D] built-in [Variant] type is a 3×4 matrix representing a transformation in 3D space. It contains a [Basis], which on its own can represent rotation, scale, and shearing. Combined with the [member origin], it can also represent a translation.
```

Comment:
```suggestion
		?? The [Transform3D] built-in [Variant] type is a 3×4 matrix representing a transformation in 3D space. It contains a [Basis], which on its own can represent rotation, scale, and shear. Combined with the [member origin], it can also represent a translation.
```
Use same form of the words, i.e. "rotating, scaling, and shearing"